stdlib/betaTransformations.hk

```diff
@@ -0,0 +1,14 @@
+def invBeta(a prob, b prob):
+	X <~ beta(a,b)
+	return X / (1 - X)
+
+def stdUniform():
+	X <~ beta(1,1)
+	return X
+
+def arcsin():
+	p <~ beta(0.5, 0.5)
+	return p
+
+
+invBeta(2,2)
```

Comment:
The standard library should contain no raw calls to distributions - it should only contain definitions.